Sources/Swift/Tools/UIRedactBuilder.swift

```diff
@@ -141,46 +135,64 @@ class UIRedactBuilder {
         return image.imageAsset?.value(forKey: "_containingBundle") == nil
     }
     
-    private func mapRedactRegion(fromView view: UIView, to: CALayer, redacting: inout [RedactRegion], area: CGRect, redactText: Bool, redactImage: Bool) {
-        let rectInWindow = (view.layer.presentation() ?? view.layer).convert(view.bounds, to: to)
-        guard (redactImage || redactText) && area.intersects(rectInWindow) && !view.isHidden && view.alpha != 0 else { return }
+    private func mapRedactRegion(fromView view: UIView, to: CALayer, redacting: inout [RedactRegion], area: CGRect, redactText: Bool, redactImage: Bool, transform: CGAffineTransform) {
```

Comment:
`Area` its not always derived from the current view, it could be from a parent view.

> Likewise, instead of passing in redactText and redactImage, why not just pass in SentryRedactOptions
Preferences

Removing `to`, not used in the last implementation.

Reply:
You're closer, but you still pass in `view` and `view.frame` separately, why not just pass in `view` and let the implementation access the frame?
![image](https://github.com/user-attachments/assets/53318e23-f8ec-48b4-8740-624a09b709a3)

Reply:
Due to the recursive nature of this function, the original frame will not be retained in subsequent calls.

Reply:
OK, I see what is going on now. That same frame is passed to all subsequent calls to the recursive function that traverse `view.subviews`. I think this is why I was confused about the branch that clears the redaction list before, I thought this was also using the frame of each subview.

Thanks for renaming this from `area` to `rootFrame`, makes it more obvious now at the callsite 👍🏻